prow/github/client.go

```diff
@@ -2430,3 +2430,79 @@ func (s *reloadingTokenSource) Token() (*oauth2.Token, error) {
 		AccessToken: string(s.getToken()),
 	}, nil
 }
+
+// GetOrgProjects returns the list of projects in this org.
+//
+// See https://developer.github.com/v3/projects/#list-organization-projects
+func (c *Client) GetOrgProjects(org string) ([]Project, error) {
```

Comment:
Does `GET /orgs/:org/projects` get all projects under each repo of an organization, or just the projects on the organization level?

Reply:
`GET /orgs/:org/projects` only gets the projects at the organisation level. I've added another function to use `GET /repos/:owner/:repo/projects` to get repo level projects as described here: https://developer.github.com/v3/projects/#list-repository-projects.